lib/worker.js

```diff
@@ -1,24 +1,31 @@
 const { resolve } = require('path')
-
+const Log = require('gk-log')
 const _ = require('lodash')
 const Promise = require('bluebird')
 
 const env = require('./env')
 const rollbar = require('./rollbar')
 const statsd = require('./statsd')
+const dbs = require('./dbs')
 
 const dev = env.NODE_ENV === 'development'
 
 module.exports = async function worker (scheduleJob, channel, job) {
   const data = JSON.parse(job.content.toString())
   if (dev) console.log(data.name, data.type, data.action)
+  const { logs } = await dbs()
+  const accountId = Number(data.accountId) ||
+    _.get(data, 'repository.owner.id') ||
+    _.get(data, 'installation.account.id') ||
+    _.get(data, 'organization.id') ||
+    null
```

Comment:
now that `accountId` can be `null`, too https://github.com/greenkeeperio/gk-log/pull/4/files needs to be adapted to account for that.

Reply:
true: https://github.com/greenkeeperio/gk-log/pull/5